src/library_pthread.js

```diff
@@ -104,6 +104,8 @@ var LibraryPThread = {
       // things.
       PThread['receiveObjectTransfer'] = PThread.receiveObjectTransfer;
       PThread['threadInitTLS'] = PThread.threadInitTLS;
+      PThread['receiveMessageRelayPort'] = PThread.receiveMessageRelayPort;
+      PThread['closeMessageRelayPort'] = PThread.closeMessageRelayPort;
```

Comment:
We should probably figure out a way to do this that isn't do hacky (one day).

Reply:
How about just calling these `messageRelayClose` and `messageReleyReceive`?   (Probably just my person preference for shorter names so feel free to ignore)

Reply:
Agree that shorter names would be nicer, but I think it's useful to clarify that we're receiving the port and not an actual message from the relay.